examples/common_v2/definitions/has_compile_errors/view_with_semi_colon_at_end.sqlx

```diff
@@ -0,0 +1,2 @@
+config { type: "view", hermetic: true }
+select 1 as test;
```

Comment:
nit: newline at eof

Reply:
I was following the pattern in that project, none of the SQLX files there have new lines. Better to stay consistent?